contents/docs/product-analytics/trends.mdx

```diff
@@ -38,13 +85,23 @@ Using the side menu, navigate to the [Product Analytics](https://us.posthog.com/
 
 You should now see a screen that looks like this
 
-![new insight](../../images/docs/user-guides/trends/new-insight.png)
+<ProductScreenshot
+    imageLight = {newInsightsLight} 
+    imageDark = {newInsightsDark}
+    classes="rounded"
+    alt="New insight"
+/>
 
 #### Adding events
 
 Now that you've created a Trends insight, the next thing is select the events you would like to see. Click the dropdown to the right of the <div class="inline-flex items-center justify-center rounded-full border-2 font-semibold" style="width: 20px; height: 20px; border-color: rgb(29, 74, 255); color: rgb(29, 74, 255); background-color: rgba(29, 74, 255, 0.15);">A</div> icon.
 
-![selecting our event](../../images/docs/user-guides/trends/select-event.png)
+<ProductScreenshot
```

Comment:
Something very funky is going on here? Maybe to do with the styling above?

![CleanShot 2024-03-18 at 11 25 59](https://github.com/PostHog/posthog.com/assets/34755028/0c549f19-426a-40a6-9449-729abfbe5b4c)